lib/cryptoexchange/exchanges/crypto_bridge/services/market.rb

```diff
@@ -0,0 +1,48 @@
+module Cryptoexchange::Exchanges
+  module CryptoBridge
+    module Services
+      class Market < Cryptoexchange::Services::Market
+        class << self
+          def supports_individual_ticker_query?
+            false
+          end
+        end
+
+        def fetch
+          output = super ticker_url
+          adapt_all(output)
+        end
+
+        def ticker_url
+          "#{Cryptoexchange::Exchanges::CryptoBridge::Market::API_URL}/ticker"
+        end
+
+        def adapt_all(output)
+          output.map do |pair|
+            base, target = pair['id'].split('_')
+            market_pair = Cryptoexchange::Models::MarketPair.new(
+                base: base,
+                target: target,
+                market: CryptoBridge::Market::NAME
+            )
+            adapt(pair, market_pair)
+          end
+        end
+
+        def adapt(output, market_pair)
+          ticker = Cryptoexchange::Models::Ticker.new
+          ticker.base      = market_pair.base
+          ticker.target    = market_pair.target
+          ticker.market    = CryptoBridge::Market::NAME
+          ticker.ask       = NumericHelper.to_d(output['ask'])
+          ticker.bid       = NumericHelper.to_d(output['bid'])
+          ticker.last      = NumericHelper.to_d(output['last'])
+          ticker.volume    = NumericHelper.to_d(output['volume'])
```

Comment:
@Swingcloud Double check the volume, it should be in the base currency in which a divide (volume/last) is needed

Reply:
sure let me fix it!